clang/Util.d

```diff
@@ -165,3 +165,11 @@ string namedTempDir(string prefix)
 
     return path.idup;
 }
+
+string asAbsNormPath(string path)
+{
+    import std.array;
+    import std.path;
+
+    return path.asAbsolutePath.asNormalizedPath.array.idup;
```

Comment:
I think you're suppose to use `to!(string)` when converting a range to a string.